docs/core/tools/dotnet-pack.md

```diff
@@ -102,6 +102,13 @@ Defines the value for the `$(VersionSuffix)` MSBuild property in the project.
 `-v|--verbosity <LEVEL>`
 
 Sets the verbosity level of the command. Allowed values are `q[uiet]`, `m[inimal]`, `n[ormal]`, `d[etailed]`, and `diag[nostic]`.
+> [!NOTE]
+> Web projects are not packable by default. To ovverride the default behaviour, add the below property to your `csproj` file
```

Comment:
- are not --> aren't
- `csproj` --> *.csproj*
- @KathleenDollard I wonder if the 1st sentence should say "Web projects using the Microsoft.NET.Sdk.Web SDK...". Maybe it's unnecessary to get that specific. 🤔 Thoughts?